### DISCUSSION TITLE
Show only reference without any text

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#5567] [Q&A] [answered]
In my yaml I have
```
citation: true
appendix-cite-as: display
```

The output is this (in spanish): 
```
Cómo citar
Por favor, cita este trabajo como:
Citation
```
I want only the citation to appear without the rest of the text (without Cómo citar, por favor cita este trabajo como:).

**Comment by mcanouil** [comment]
Could you share a small "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by mcanouil** [comment]
Unfortunately and after checking, it isn't currently possible to modify this from the user side.

<details><summary>See Quarto document code example.</summary>

````qmd
---
title: "Quarto Playground"
format: html
citation: true
appendix-cite-as: display
---

This is a playground for Quarto.
````

</details>

**Reply by mcanouil** [reply]
What you can do as a workaround is to tweak the text:

```yaml
language: 
  appendix-attribution-cite-as: " "
```

and customise the title as you can't remove the anchor link.

```yaml
language: 
  section-title-citation: "Something else than 'Cómo citar'"
```

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/cdf69e6f-0001-46ff-9ea1-fd57587206a2">

**Reply by mcanouil** [reply, marked as answer]
As pointed out by @dragonstyle you can use css to target the section and hide it.
To note, the id html value depends on the lang.

<table>
<tr>
<th>Quarto document</th>
<th>HTML</th>
</tr>
<tr>
<td>

````qmd
---
title: "Quarto Playground"
format: html
citation: true
appendix-cite-as: display
lang: es
language: 
  appendix-attribution-cite-as: " "
include-in-header: 
  - text: "<style>#cómo-citar { display: none; }</style>"
---

This is a playground for Quarto.
````

</td>
<td>
<img width="495" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/395437f6-d951-4979-abe4-bd49c227c58d">
</td>
</tr>
</table>
